data/json/npcs/TALK_TRUE_FOODPERSON.json

```diff
@@ -0,0 +1,550 @@
+                      { "u_has_trait": "PROF_FOODP" },
+                      { "or": [ { "u_is_wearing": "foodperson_mask" }, { "u_is_wearing": "foodperson_mask_on" } ] }
+                    ]
+                  }
+                ]
+              }
+            }
+          ]
+        },
+        "trial": { "type": "PERSUADE", "difficulty": 20, "mod": [ [ "TRUST", 2 ], [ "VALUE", 2 ] ] },
+        "success": { "topic": "TALK_FOODPERSON_SHELTER" },
+        "failure": { "topic": "TALK_FOODPERSON_NOSHELTER" }
+      },
+      {
+        "text": "I want to get stronger.  Please teach me.",
+        "condition": {
+          "and": [
+            { "not": { "npc_has_var": "npc_failed_training", "type": "dialogue", "context": "foodperson", "value": "yes" } },
+            {
+              "not": { "npc_has_var": "npc_succes_training", "type": "dialogue", "context": "foodperson", "value": "yes" }
+            },
+            {
+              "not": {
+                "and": [
+                  { "npc_has_var": "npc_foodpersons_sharing", "type": "dialogue", "context": "foodperson", "value": "yes" },
+                  {
+                    "or": [
+                      { "u_has_trait": "PROF_FOODP" },
+                      { "or": [ { "u_is_wearing": "foodperson_mask" }, { "u_is_wearing": "foodperson_mask_on" } ] }
+                    ]
+                  }
+                ]
+              }
+            }
+          ]
+        },
+        "trial": { "type": "PERSUADE", "difficulty": 20, "mod": [ [ "TRUST", 2 ], [ "VALUE", 2 ] ] },
+        "success": { "topic": "TALK_FOODPERSON_TRAINING" },
+        "failure": { "topic": "TALK_FOODPERSON_NOTRAINING" }
+      },
+      {
+        "text": "Please share your knowledge.",
+        "condition": {
+          "and": [
+            { "not": { "npc_has_var": "npc_failed_training", "type": "dialogue", "context": "foodperson", "value": "yes" } },
+            {
+              "or": [
+                { "npc_has_var": "npc_succes_training", "type": "dialogue", "context": "foodperson", "value": "yes" },
+                {
+                  "npc_has_var": "npc_foodperson_impressed",
+                  "type": "dialogue",
+                  "context": "foodperson",
+                  "value": "yes"
+                },
+                {
+                  "and": [
+                    { "npc_has_var": "npc_foodpersons_sharing", "type": "dialogue", "context": "foodperson", "value": "yes" },
+                    {
+                      "or": [
+                        { "u_has_trait": "PROF_FOODP" },
+                        { "or": [ { "u_is_wearing": "foodperson_mask" }, { "u_is_wearing": "foodperson_mask_on" } ] }
+                      ]
+                    }
+                  ]
+                }
+              ]
+            }
+          ]
+        },
+        "topic": "TALK_TRAIN"
+      },
+      {
+        "text": "I'm building up a team, people with special abilities.",
```

Comment:
```suggestion
        "text": "I'm building up a team, I need people with special abilities.",
```

Reply:
So that's a direct quote from the justice league movie ^^". Maybe it's the punctuation that is wrong, I should use `:` instead of `,` ?

Reply:
No, leave it as is then.